scripts/files/tests/fs_s3_test.py

```diff
@@ -124,3 +124,20 @@ def test_exists_object_starting_with_not_exists() -> None:
     file_exists = exists("s3://testbucket/hello/another.fi")
 
     assert file_exists is False
+
+
+@mock_s3  # type: ignore
+def test_list_files_in_uri() -> None:
```

Comment:
A [slightly different pattern](https://github.com/linz/geostore/blob/master/tests/test_import_dataset.py#L84) might be useful to get type annotations for your S3 clients.

Reply:
Would be better to do that in a separate PR.